game/views.py

```
@@ -1571,6 +1603,35 @@ def _resolve_lesson_name(url_key):
+                lesson in completed_lessons
+                for lesson in previous_level["lessons"]
+            )
+
+        if not previous_level_complete:
+            continue
+
+        for i, lesson in enumerate(lessons):
+            if i == 0:
+                unlocked.add(lesson)
+            elif lessons[i - 1] in completed_lessons:
+                unlocked.add(lesson)
```

Ruff (0.15.15)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)